src/main/java/com/stripe/model/OrderItem.java

```diff
@@ -1,11 +1,13 @@
 package com.stripe.model;
 
-public class OrderItem extends StripeObject {
+import com.stripe.net.APIResource;
```

Comment:
I had to make `OrderItem` extend `APIResource` instead of `StripeObject` so it can use `ExpandableField`. Technically, order_items are a subresource and not a first-order API resource, but I don't think there's any ill effect to making it extend `APIResource`.

That said, maybe we ought to move `ExpandableField` so that it's usable from `StripeModel` instead?